index.bs

```diff
@@ -10222,46 +10222,38 @@ See [[#es-DOMString]] for the specific requirements that the use of [{{TreatNull
 
 <h4 id="Unforgeable" extended-attribute lt="Unforgeable">[Unforgeable]</h4>
 
-If the [{{Unforgeable}}]
-[=extended attribute=]
-appears on a non-[=Static attributes|static=]
-[=attribute=]
-or non-[=static operations|static=]
-[=operations=], it indicates
-that the attribute or operation will be reflected as an ECMAScript property in
-a way that means its behavior cannot be modified and that performing
-a property lookup on the object will always result in the attribute’s
-property value being returned.  In particular, the property will be
-non-configurable and will exist as an own property on the object
-itself rather than on its prototype.
+If the [{{Unforgeable}}] [=extended attribute=] appears on
+[=regular attributes=]or non-[=static operations|static=] [=operations=],
```

Comment:
You can switch out the non-static operations for regular operations as well. Seems like that change (but for attributes) was the only one done in this PR. Also there should be a space between `=]` and `or`.